pkg/models/init_client.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+// InitClient model
+// swagger:model Init
+type InitClient struct {
```

Comment:
what are the pros/cons of this new model over the version model?

Reply:
The response isn't returning a full `Version` model. One reason for having a model is that it can be changed from a central location instead of within the endpoint.

The main reason though is how would we return a `Version` with only a select number of its properties? If there are properties in the `Version` model we don't want we would probably have to do something like:

```go
verson.PropOne = nil
version.PropThree = nil
```

I'm open to feedback on this, above way could be better but I think it's less clean.

Reply:
Do you see any cons with using the existing Version model with `omitempty` on all the unwanted properties?